docs/managed_package.rst

```diff
@@ -1,27 +1,156 @@
+
+
+
+Create a Beta Version
+---------------------
+The ``release_beta`` flow exists to group the common tasks that we want
+to execute when we release a new beta version of a project.
+
+.. code-block:: console
+
+    $ cci flow run release_beta --org packaging
+
+It is typically run against the project's packaging org, and does the following: 
+
+* Uploads new beta version to the packaging org
+* Creates a new GitHub release tag for the new beta version
+* `Generates Release Notes`_
+* Syncs feature branches with the ``main/`` branch.
+
+If you just want to create a new beta version for your project,
+without the bells and whistels, you can just use the ``upload_beta`` task:
+
+.. code-block:: console
+
+    $ cci task run upload_beta --name package_version 
+
+
+
+Test a Beta Version
+-------------------
+Test your latest new Beta version with:
+
+.. code-block:: console
+
+    $ cci flow run ci_beta --org <TODO> 
+
+This flow installs the latest beta version of the project into a scratch org, and executes tests against it.
+This flow is intended to be run every time a beta release is created.
 
-Upload and test a beta version
-------------------------------
 
-Generate release notes
+
+Generate Release Notes
 ----------------------
+The ``github_release_notes`` task fetches the text from pull requests that
+were merged between two given tags. The task then searches for specific titles
+(Critical Changes, Changes, Issues Closed, New Metadata, Installation Info) in
+the pull request bodies and aggregates the text together under those titles in
+in the description of a GitHub tag.
+
+You can view what the release notes would look like without publishing them to GitHub:
+
+.. code-block::
+
+    $ cci task run github_release_notes --tag release/1.2
+
+This would aggregate text from pull requests between releases 1.2 and next most recent release.
+You can also see where each line in the release notes comes from from by using the ``--link_pr True`` option.
+
+If you want to publish these release notes to a release tag in GitHub use the ``--publish`` option:
+
+.. code-block::
+
+    $ cci task run github_release_notes --tag release/1.2 --publish True
+
+
+If your team wants to use additional headings you can add new ones by
+putting the following under the ``project`` -> ``git`` section of your ``cumulusci.yml`` file:
+
+.. code-block::
+
+    release_notes:
+        parsers:
+            7: class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
+
+.. note::
+
+    The new parser is listed with the number ``7`` because the first six are the
+    `default parsers <https://github.com/SFDO-Tooling/CumulusCI/blob/671a0e88cef79e9aeefe1e2b835816cd8141bdbb/cumulusci/cumulusci.yml#L1154>`_ that come with CumulusCI.
+        
 
-Upload and test a final version
+
+Upload and Test a Final Version
 -------------------------------
+When you're ready to upload a production release of your Managed Package project use the ``--production True`` option:
```

Comment:
This is confusing because this option is not actually part of the command shown here (it's configured as part of the release_production flow)